src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Globalization.cs

```diff
@@ -40,7 +40,7 @@ internal static unsafe partial class Kernel32
         [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
         internal static extern int LocaleNameToLCID(string lpName, uint dwFlags);
 
-        [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
>SetLastError = true [](start = 62, length = 19)

should SetLastError = true be ifdefed for the Debug build? according to Jan it will add some perf cost and we are not going to use it anyway in the release build